src/project/inotationproject.h

```diff
@@ -58,8 +58,10 @@ class INotationProject
 
     virtual void markAsUnsaved() = 0;
 
+    virtual void revertChanges() = 0;
```

Comment:
It looks like we're not using this method because it's fully handled in `ProjectActionsController::revertCorruptedScoreToLastSaved`.